packages/test/test-end-to-end-tests/src/test/gc/gcDatastoreAliased.spec.ts

```diff
@@ -79,9 +79,7 @@ describeFullCompat("GC Data Store Aliased Full Compat", (getTestObjectProvider)
 		const containerRuntime2 = mainDataStore2._context
 			.containerRuntime as unknown as IContainerRuntime;
 		assert.doesNotThrow(
-			async () =>
-				containerRuntime2.getAliasedDataStoreEntryPoint?.(alias) ??
-				containerRuntime2.getRootDataStore(alias),
```

Comment:
So even for compat tests we don't need this fallback anymore?  Can we delete `getRootDataStore`?

Reply:
I believe we can. It was deprecated in the last release.

Reply:
Maybe I will do that in a separate PR since there is a slim chance of that being backed out for some reason.